src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -50,4 +55,10 @@
     text-overflow: ellipsis;
     white-space: nowrap;
   }
+
+  .pf-c-chip {
+    &:not(:last-child):not(:only-child) {
+      margin-right: var(--pf-c-chip--MarginRight);
```

Comment:
Can you rename this var and the `pf-c-chip` vars on L21-23?

https://github.com/patternfly/patternfly-next/blob/13cfdaa1aed879152dcd25096972e424e01f6002/src/patternfly/components/ChipGroup/chip-group.scss#L21-L23